### ISSUE TITLE
Ship transfer arrival script error

### ISSUE THREAD

**Issue by Darkcyde13** [EDCD/EDDI#257]
## Steps to reproduce

Initiate a ship transfer and wait for it to arrive.

## Expected
Speech should say (for example):
"Warrant Officer, Your Asp has completed its transfer to Gaiman Station in the Akandinigua system."

## Observed
Script actually says:
"Your Asp has completed its transfer 'to') Tiliala's Lament in the Eurybia system."

There is **'to')** in the middle.

## Investigation
I discovered this when transferring my Asp as in the observed example.  I looked over the script and found the problem.  There is unnecessary use of single quotes and escaping of them, plus the extra close bracket.

I have tidied up the script and made the second part a little more efficient:-

```
{Occasionally(2, "{F('Honorific')}, ")}
Your {event.ship} has 

{OneOf("arrived {if event.system && event.station: at}", 
       "completed its transfer {if event.system && event.station: to}")}

{if event.station:
   {if event.system != system.name && event.system:
      {event.station} in the {event.system} system
   |elif event.station != station.name:
      {event.station}
   }
}.
```

**Comment by Darkcyde13** [comment]
I was just reading through this to make sure I'd got it all correct, when I realised that in the observed speech, it had used the system the ship was transferred **_from_**, not **_to_**.  This was using the default script.

However, after I edited the script and transferred my ship to Akandinigua (as in the expected speech) it was correct.  I'll try this again and see what happens.

**Comment by Darkcyde13** [comment]
It just occurred to me that the Module Arrived event was also added in the new version of EDDI, so I checked it out.

While I have not tested the actual transfer of a module, the script suffers from the same errors mentioned above in the Ship Arrived event script.  It would appear these were 'copy & pasted' when the two events were created.  The same fixes also apply to this event as well.

Would you like me to open a new issue for this one too?

**Comment by Tkael** [comment]
This should be addressed by PR #264. Thanks for reporting. :-)
